src/helpers/makeA11yQuery.js

```diff
@@ -1,12 +1,17 @@
 // @flow
 import waitFor from '../waitFor';
+import { getQueriesForElement } from '../within';
 import type { WaitForOptions } from '../waitFor';
 import {
   ErrorWithStack,
   prepareErrorMessage,
   createQueryByError,
 } from './errors';
 
+type QueryOptions = {
```

Comment:
The name in `QueryOptions` type for `getBy` is optional. But, it's a must for the function, `filterWithName`. Is this duplication that I get rid of or this okay?
(I haven't worked with types all that much 😬 )